integration_tests/suite/helpers/auth.py

```diff
@@ -24,12 +24,17 @@ def make_token(
         user_uuid=None,
         tenant_uuid=TENANT1_UUID,
         session_uuid='my-session-uuid',
-        acl=['websocketd']
+        acl=['websocketd'],
+        purpose='user',
     ):
-        metadata = {'tenant_uuid': str(tenant_uuid)}
+        metadata = {'tenant_uuid': str(tenant_uuid), 'purpose': purpose}
+
         if token_uuid is None:
             token = MockUserToken.some_token(
-                session_uuid=str(session_uuid), acl=acl, metadata=metadata
+                user_uuid=str(user_uuid),
```

Comment:
But default user_uuid is `None` ... What will be the effect to have `user_uuid='None'` in tests?
Is it possible to be more explicit?

Reply:
Made an alias USER1_UUID used by default, to avoid getting a None anywhere.  Also simplfies the code